.gitmodules

```diff
@@ -31,7 +31,7 @@
 	branch = master
 [submodule "contrib/client-c"]
 	path = contrib/client-c
-	url = https://github.com/tikv/client-c.git
+	url = https://github.com/hongyunyan/client-c.git
```

Comment:
Why change this submodule here?

Reply:
we need to get this PR merged first: https://github.com/tikv/client-c/pull/146

Reply:
At the beginning of the submission, CI could not pull the correct version of client-c, so I made this change. Anyway  after the code review for client-c completed and merged, it will be changed back.